app/controllers/rentals_controller.rb

```diff
@@ -26,7 +26,7 @@ def show
   private
 
   def rental_params
-    {user_id: session[:user_id], days_rented: session[:cart]['days'], total_price: @cart.total_price, status: 'order placed'}
+    {user_id: session[:user_id], days_rented: session[:cart]['days'], total_price: @cart.total_price, status: 'Ordered'}
```

Comment:
Line is too long. [120/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.